packages/prisma/seed-app-store.ts

```diff
@@ -251,6 +251,12 @@ export default async function main() {
       client_secret: process.env.ZOHOCRM_CLIENT_SECRET,
     });
   }
+  if (process.env.ZOHO_BIGIN_CLIENT_ID && process.env.ZOHO_BIGIN_CLIENT_SECRET) {
+    await createApp("zoho-bigin", "zoho-bigin", ["other"], "zoho-bigin_other_calendar", {
+      client_id: process.env.ZOHO_BIGIN_CLIENT_ID,
+      client_secret: process.env.ZOHO_BIGIN_CLIENT_SECRET,
+    });
+  }
```

Comment:
Not needed. We won't seed any new app. admin/apps UI would control this.

Reply:
@hariombalhara how can I access this admin/apps UI where I can plug the secrets?